packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1093,11 +1092,11 @@ new PrismaClient({
           // if this `next` was called and there's some more middlewares
           const nextMiddleware = this._middlewares.query.get(++index)
 
-          if (nextMiddleware) {
-            // we pass the modified params down to the next one, & repeat
-            return nextMiddleware(changedParams, consumer)
-          }
+          // we pass the modified params down to the next one, & repeat
+          if (nextMiddleware) return nextMiddleware(changedParams, consumer)
+          // calling `next` calls the consumer again with the new params
```

Comment:
```suggestion
          // we pass the modified params down to the next one, & repeat
          // calling `next` calls the consumer again with the new params
          if (nextMiddleware) return nextMiddleware(changedParams, consumer)
```
then this might be easier to read